### ISSUE TITLE
Target apply [question]

### ISSUE THREAD

**Issue by yurii-kryvosheia** [leg100/pug#136]
Is there a method to achieve the same outcome as `terraform plan -target module.tests` without selecting each resource in a state?

**Comment by leg100** [comment]
Do you mean you want to be able to select all resources in module `module.tests` without having to individually select each resource?

If so, you could achieve this by filtering by `module.tests`. (see readme for info on filtering).

**Comment by yurii-kryvosheia** [comment]
I probably missed the feature release that allowed for `Ctrl+A` selection. That should do the trick. Thanks.

**Comment by leg100** [comment]
@yurii-kryvosheia Great. However, having had a chance to reflect on this, what I have suggested is somewhat different to what you originally asked for:

1) `terraform plan -target module.tests` creates a plan for _both existing and yet-to-exist_ resources in `module.tests`.

2) Whereas what I've suggested runs `terraform plan -target module.tests.resource1 module.tests.resource2 ...`, and therefore only creates a plan for the _existing_ resources, i.e. `module.tests.resource1`, `module.tests.resource2`, etc.

Do keep this in mind.

In the meantime I'll see if I can add support for 1).
